src/components/main/note/Note.tsx

```diff
@@ -246,14 +251,17 @@ const Note = ({ note, id, userName, scale }: NoteProps): JSX.Element => {
               borderColor: isTransforming ? lightBlue[500] : 'none',
             }}
             onClick={handleClickEvent}
+            data-cy={`${NOTE_CY}-${id}`}
```

Comment:
If you find it easier, you can have a utility function that build this id for you, so you are sure you always build it the same regardless.